src/components/Public/Page/Attachment/Uploader.vue

```diff
@@ -102,9 +109,24 @@ export default {
 
     onSuccess (file, { response }) {
       this.active = null
+      this.error = null
       this.$emit('uploaded', response, file)
     },
 
+    onFileAdded (file) {
+      this.error = null
+
+      // Check if file type is allowed
+      if (!this.acceptedFiles.includes(file.type) && this.acceptedFiles.length > 0) {
+        this.error = this.$t('general.label.fileTypeNotAllowed')
+        this.$refs.dropzone.removeFile(file)
+      }
```

Comment:
How about `image/*` -- this won't work with this code.
Also -- `file.type` can be misleading.

Iirc some operating systems can define custom mime types based on the application that will open that file. So for example if you upload a .csv and excel is set to open it, it's mime type won't be `text/csv`, but will be `application/what_ever_excel_has`.

You could guess the type from the extension, or guess it from the content (https://golangcode.com/get-the-content-type-of-file/ -- go example, but in theory, it's the same)

Reply:
https://github.com/cortezaproject/corteza-webapp-compose/commit/4ed34dd57f5009c5f89cbe0701ab67e6c2f7ea59#diff-7c0a8b0cf4b7322d1666cabfc4219c44R123

Reply:
This works for images/* and similar.
Best i can do, feel free to improve.